scipy/cluster/hierarchy.py

```diff
@@ -3093,6 +3093,8 @@ def dendrogram(Z, p=30, truncate_mode=None, color_threshold=None,
         -sized list (or tuple). The ``labels[i]`` value is the text to put
         under the :math:`i` th leaf node only if it corresponds to an original
         observation and not a non-singleton cluster.
+
+        Note, Dimensions of ``Z`` and ``labels`` must be consistent.
```

Comment:
minor: could you spell dimensions with a lower-case D?

Reply:
Actually, it would be better to spell out what `n` is in the paragraph above. E.g. change `"n-sized list (or tuple)"` to
```
`n`-sized sequence, with ``n == Z.shape[1]``
```